server/models/LegalDocument.js

```diff
@@ -4,12 +4,14 @@ export const LEGAL_DOCUMENT_TYPE = {
   US_TAX_FORM: 'US_TAX_FORM',
 };
 
-function defineModel() {
-  const NOT_REQUESTED = 'NOT_REQUESTED';
-  const REQUESTED = 'REQUESTED';
-  const RECEIVED = 'RECEIVED';
-  const ERROR = 'ERROR';
+export const LEGAL_DOCUMENT_REQUEST_STATUS = {
+  NOT_REQUESTED: 'NOT_REQUESTED',
+  REQUESTED: 'REQUESTED',
+  RECEIVED: 'RECEIVED',
+  ERROR: 'ERROR',
+};
```

Comment:
When should we have constants here versus in the `constant` folder?

Reply:
I'm not sure, arguably all shared constants should go in the constants folder? Would be great to define some rules around this.